src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -16,12 +19,21 @@ test.describe("Buzz Page", () => {
     await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
   });
 
-  test("Test Case 1: Share file and Delete file", async ({ page }) => {
+  test("Test 1: Post Should Be Shared and Deleted", async ({ page }) => {
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePhotos();
+    await buzzPage.sharePhotos(filePath);
     await page.reload();
     await expect(buzzPage.photoImg.first()).toBeAttached();
     await buzzPage.deletePhotos();
     await expect(buzzPage.noPostParagraph.first()).toHaveText(buzzPageAssertions.noPostAvailable);
   });
+
+  test("Test 2: Post Should Be Edited", async ({ page }) => {
+    await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
+    await buzzPage.sharePhotos(filePath);
+    await page.reload();
+    await buzzPage.editPost(postEdited)
+    await expect(page.getByText(postEdited)).toBeAttached();
+    await buzzPage.deletePhotos();
```

Comment:
I think you should think of moving this line to afterEach - it should work with empty photos tho not to fail after "deleting" test.